core-bundle/src/Resources/config/services.yml

```diff
@@ -1039,14 +1046,6 @@ services:
 
     Contao\CoreBundle\Framework\ContaoFrameworkInterface: '@contao.framework'
```

Comment:
This should be deprecated, we don't use that interface. Not sure it was ever available for autowiring?

Reply:
Yes, it was.

https://github.com/contao/contao/blob/904fd1846abbb051783b8b6d043abaffab3553bb/core-bundle/src/Resources/config/services.yml#L29-L30

I have just moved it down into the "backwards compatibility" section now.